contracts/fee_splitter/src/testing.rs

```diff
@@ -1 +1,144 @@
+use bindings_test::mock_coreum_deps;
+use cosmwasm_std::{
+    testing::{mock_env, mock_info, MOCK_CONTRACT_ADDR},
+    Attribute, BankMsg, Coin, CosmosMsg, Decimal, ReplyOn, Uint128,
+};
 
+use crate::{
+    contract::{execute, instantiate, SubMsg},
+    error::ContractError,
+    msg::{ExecuteMsg, InstantiateMsg},
+};
+
+#[test]
+fn init_works() {
+    let mut deps = mock_coreum_deps(&[]);
+    let env = mock_env();
+    let sender = "addr0000";
+    let info = mock_info(sender, &[]);
+
+    let first_tupple = ("ATOM".to_string(), Decimal::percent(50u64));
+    let second_tuple = ("TIA".to_string(), Decimal::percent(50u64));
+    let msg = InstantiateMsg {
+        addresses: vec![first_tupple.clone(), second_tuple.clone()],
+        cw20_contracts: vec!["USDT".to_string()],
+    };
```

Comment:
This is still wrong. Why are there denoms there?

Reply:
fixed